ansible/roles/os_zabbix/tasks/main.yml

```diff
@@ -70,6 +70,10 @@
   tags:
   - openshift_logging
 
+- include_vars: template_ops_sso.yml
+  tags:
+  - sso
```

Comment:
I currently have some `sso` tags and some `sso_app` tags. Currently, the distinction is, if it's setting up the app (like this), it should use `sso_app`, and if it's setting up the AWS side, `sso`. But maybe we should just use `sso_app` across the board.